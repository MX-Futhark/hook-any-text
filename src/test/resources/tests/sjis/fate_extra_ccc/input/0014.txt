82 E0 82 BF 82 EB 82 F1 81 41 88 DF 91 95 91 E3 82 CD 82 BB 82 BF 82 E7 8E 9D 82 BF 82 C5 EA 58 0A 89 C2 88 A4 82 A2 95 9E 88 C8 8A 4F 82 CD 8F C1 82 B5 92 59 82 C9 82 B5 82 DC 82 B7 82 CC 82 C5 81 41 0A 82 BB 82 BF 82 E7 82 CC 95 FB 82 E0 82 B2 92 8D 88 D3 82 CC 82 D9 82 C7 82 F0 81 42 00 00 00 00 00 00 00 00 00 01 00 00 00 02 00 00 00 03 00 00 00 04 00 00 00 05 00 00 00 06 00 00 00 07 00 00 00 08 00 00 00 09 00 00 00 0A 00 00 00 0B 00 00 00 0C 00 00 00 0D 00 00 00 0E 00 00 00 0F 00 00 00 10 00 00 00 11 00 00 00 12 00 00 00 13 00 00 00 14 00 00 00 15 00 00 00 16 00 00 00 17 00 00 00 18 00 00 00 19 00 00 00 1A 00 00 00 1B 00 00 00 1C 00 00 00 1D 00 00 00 1E 00 00 00 1F 00 00 00 20 00 00 00 21 00 00 00 22 00 00 00 23 00 00 00 24 00 00 00 25 00 00 00 26 00 00 00 27 00 00 00 28 00 00 00 29 00 00 00 2A 00 00 00 2B 00 00 00 2C 00 00 00 2D 00 00 00 2E 00 00 0B 00 00 00 F8 14 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 00 00 5C 00 00 00 00 00 00 00 88 43 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 50 43 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 60 C1 FF FF FF FF 00 00 00 00 00 00 88 43 00 00 00 00 FF FF FF DC 00 00 0A 43 FF FF FF FF 00 00 50 43 00 00 88 43 00 00 18 43 FF FF FF DC 0B 00 00 00 78 15 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 01 00 5C 00 00 00 00 00 00 80 9B 43 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 50 43 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 00 C0 FF FF FF FF 00 00 00 00 00 00 88 43 00 00 00 00 FF FF FF DC 00 00 FC 42 FF FF FF FF 00 00 50 43 00 00 88 43 00 00 00 43 FF FF FF DC 02 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 02 00 C2 01 00 00 00 00 90 12 A3 48 0B 00 00 00 98 17 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 02 00 10 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 00 00 00 00 3C 43 00 00 00 00 00 00 F0 43 00 00 00 00 FF FF FF DC 00 00 F0 43 00 00 3C 43 00 00 00 00 00 00 00 00 00 00 A8 42 FF FF FF DC 00 00 00 00 00 00 88 43 00 00 00 00 00 00 F0 43 00 00 A8 42 FF FF FF DC