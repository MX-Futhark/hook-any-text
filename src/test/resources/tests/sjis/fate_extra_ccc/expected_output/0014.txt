["もちろん、衣装代はそちら持ちで♥
可愛い服以外は消し炭にしますので、
そちらの方もご注意のほどを。"]